packages/common/container-definitions/src/tokenProvider.ts

```diff
@@ -3,12 +3,21 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * @deprecated Fluid Framework does not prescribe a particular approach to token providers.
```

Comment:
Could we add the version that it's being deprecated in for these comments?

Reply:
Added in latest